packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -16,6 +16,43 @@ import '../cache.dart';
 import '../globals.dart';
 import 'sdk.dart';
 
+/// Represents Flutter-specific data that is added to the `PUB_ENVIRONMENT`
+/// environment variable and allows understanding the type of requests made to
+/// the package site on Flutter's behalf.
+// DO NOT update without contacting kevmoo.
+// We have server-side tooling that assumes the values are consistent.
+class PubContext {
```

Comment:
Can you clarify why we need these to be so specific (and thus less 'stable')? Wouldn't it be enough to differentiate between package commands that are from developers, and commands that are from our internal tooling?

Reply:
the 'update_packages' bits cover a lot of the things we were seeing, but I'm also seeing `barback` coming through w/ a direct dependency via verify, create, etc.

I want to understand which commands are doing that.